Junit/src/IPL.java

```diff
@@ -0,0 +1,364 @@
+import java.util.*;
+import java.io.*;
+import java.util.stream.*;
+
+
+class Player implements Comparable<Player>{
+
+    String name;
+    String team;
+    String role;
+    int matches;
+    int runs;
+    double average;
+    double strikeRate;
+    int wickets;
+
+    Player(String name,String team,String role,int matches,int runs,double average,double strikeRate,int wickets){
+        this.name = name;
+        this.team = team;
+        this.role = role;
+        this.matches = matches;
+        this.runs  =runs;
+        this.average = average;
+        this.strikeRate  = strikeRate;
+        this.wickets = wickets;
+    }
+
+
+
+    public String getName(){
+        return name;
+    }
+
+    public String getRole(){
+        return role;
+    }
+
+    public int getWicket(){
+        return wickets;
+    }
+
+    public String getTeam(){
+        return team;
+    }
+
+    public int getRuns(){
+        return runs;
+    }
+
+    public int compareTo(Player p){
+        return 0;
+    }
+
+    // for all rounders
+    public int cumulativeScore(){
+        return wickets*40 + runs;
+    }
+
+    public int getMatches(){
+        return matches;
+    }
+}
+
+
+class WicketComparator implements Comparator<Player>{
+    public int compare(Player p1,Player p2){
+        if(p1.getWicket() < p2.getWicket()){
+            return 1;
+        }
+        else if(p1.getWicket() > p2.getWicket()){
+            return -1;
+        }
+        return 0;
+    }
+}
+
+class RunComparator implements Comparator<Player>{
+    public int compare(Player p1,Player p2){
+        if(p1.getRuns() < p2.getRuns()){
+            return 1;
+        }
+        else if(p1.getRuns() > p2.getRuns()){
+            return -1;
+        }
+        return 0;
+    }
+}
+
+
+class RunWicketComparator implements Comparator<Player>{
+    public int compare(Player p1,Player p2){
+        if(p1.cumulativeScore() < p2.cumulativeScore()){
+            return 1;
+        }
+        else if(p1.cumulativeScore() > p2.cumulativeScore()){
+            return -1;
+        }
+        return 0;
+    }
+}
+
+
+
+
+class IPL{
+
+    // for putting all players into list of player objects.
+    public static List<Player> putPlayers(){
```

Comment:
This should have been part of a separate utility class, that would have a method that takes the filename as an argument reads the data from it, and return a list of CSV objects.